analytics/js_tracking_api.rst

```diff
@@ -839,3 +839,19 @@ Example of usage::
 
     _paq.push(["trackSiteSearch", "test", false, 20]);
 
+
+
+Tracking user anonymously
+``````````````````````````
+To track visitor anonymously (without consent) use the ``setUserIsAnonymous`` function::
+
+    _paq.push(["setUserIsAnonymous", isAnonymous]);
+
+.. describe:: isAnonymous
+
+    **Required** ``integer/boolean`` Flag that sets anonymous tracking on and off (true/false or 1/0)
```

Comment:
I think that alternative types are usually separated by `|` character (e.g. in [JSDocs](http://usejsdoc.org/tags-type.html)), but I see that this character is also used in our docs.

Try to format JS values in descriptions. Check how it's done in `checkOnScroll` description in this file.

Reply:
Hmm... setEcommerceView parameters description mislead me.

Reply:
I'll keep it simple and stick with boolean.